ansible/deploy.yml

```diff
@@ -1,7 +1,23 @@
 ---
-- name: Deploy binaries
-  # Deploy on both VMs, only the primary is "active"
+- name: Build LARD binary
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Build
+      ansible.builtin.command:
+        chdir: ..
```

Comment:
Ok, this is a bit of a hack, it expects that we are inside the `ansible` directory